module3/preparation/week3/CreatingRelatedResources.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+title: Creating and Reading Related Resources Preparation
+---
+
+## Understanding Forms
```

Comment:
I like this preparation a lot! I wonder if we should make time explicitly in the lesson to talk about this exercise?

Might also be nice to include the link to the create a single resource lesson because that's going to be a big reference to use for this lesson.